dev/DynamicDependency/appmodel_msixdynamicdependency.h

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include <appmodel.h>
+
+// Modern variants of appmodel.h
+namespace appmodel
+{
+inline std::wstring GetPackagePath(PCWSTR packageFullName)
```

Comment:
Sure it does.  Things inside namespace start at same level e.g.
```
namespace Foo
{
void Bar();
...
}
```

If you find something otherwise drop a note, it's probably a mistake (probably VS being 'helpful' and reformatted code...)